__tests__/app.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require("supertest");
+const app = require("../app");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+
+beforeEach(()=> {
+    return seed(data)
+});
+
+afterAll(() => {
+      return db.end();
+});
+
+describe("app", ()=> { 
+describe("GET /api/categories", () => {
+    test("200: responds with array with properties slug and description", () => {
+        return request(app)
+        .get('/api/categories')
+        .expect(200)
+        .then(({ body }) => {
+            const {categories} = body;
+            expect(categories).toHaveLength(4);
+            expect(categories[0]).toHaveProperty("slug", expect.any(String));
+            expect(categories[0]).toHaveProperty("description", expect.any(String));
+        })
+    })
```

Comment:
This is only testing first element in the array. Use iteration to check each element is what you'd expect.

Reply:
This is only testing first element in the array. Use iteration to check each element is what you'd expect.